numcodecs/__init__.py

```diff
@@ -115,3 +115,7 @@
 
 from numcodecs.fletcher32 import Fletcher32
 register_codec(Fletcher32)
+
+with suppress(ImportError):
```

Comment:
Isn't it better to make sure this import succeeds (`import pcodec` inside `encode`/`decode`, below), but presenting a helpful message whenever someone tries to use Pcodec without the necessary deps? Otherwise, the user would just see some "codec not found" message without any idea of what to do about it.

Reply:
I agree with this comment. The approach used here was copied from other implementations, but I'm going to change it to what you suggested.